<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>assign</class>
 <widget class="QWidget" name="assign">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stack">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>750</width>
       <height>600</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="assign_page">
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>34</x>
         <y>338</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 290 12pt &quot;微软雅黑 Light&quot;;</string>
       </property>
       <property name="text">
        <string>9:00-11:00影片选择：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>42</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 290 12pt &quot;微软雅黑 Light&quot;;</string>
       </property>
       <property name="text">
        <string>日期选择：</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit">
       <property name="geometry">
        <rect>
         <x>175</x>
         <y>49</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="do_assign">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>490</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>进行排片</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>29</x>
         <y>-9</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 290 12pt &quot;微软雅黑 Light&quot;;</string>
       </property>
       <property name="text">
        <string>排片功能：</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>92</y>
         <width>631</width>
         <height>231</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="assign_check">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>490</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>查看排片结果</string>
       </property>
      </widget>
      <widget class="QPushButton" name="date_select">
       <property name="geometry">
        <rect>
         <x>502</x>
         <y>43</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>选定排片日期</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="movie1">
       <property name="geometry">
        <rect>
         <x>252</x>
         <y>345</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>353</x>
         <y>342</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 290 12pt &quot;微软雅黑 Light&quot;;</string>
       </property>
       <property name="text">
        <string>13:00-15:00影片选择：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="movie2">
       <property name="geometry">
        <rect>
         <x>591</x>
         <y>349</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>19</x>
         <y>387</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 290 12pt &quot;微软雅黑 Light&quot;;</string>
       </property>
       <property name="text">
        <string>15:00-17:00影片选择：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="movie3">
       <property name="geometry">
        <rect>
         <x>253</x>
         <y>394</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>352</x>
         <y>388</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 290 12pt &quot;微软雅黑 Light&quot;;</string>
       </property>
       <property name="text">
        <string>17:00-19:00影片选择：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="movie4">
       <property name="geometry">
        <rect>
         <x>592</x>
         <y>393</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>18</x>
         <y>428</y>
         <width>231</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 290 12pt &quot;微软雅黑 Light&quot;;</string>
       </property>
       <property name="text">
        <string>19:00-21:00影片选择：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="movie5">
       <property name="geometry">
        <rect>
         <x>253</x>
         <y>440</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>347</x>
         <y>432</y>
         <width>251</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 290 12pt &quot;微软雅黑 Light&quot;;</string>
       </property>
       <property name="text">
        <string>21:00-23:00影片选择：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="movie6">
       <property name="geometry">
        <rect>
         <x>592</x>
         <y>440</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="assign_result">
      <widget class="QTableWidget" name="tableWidget_2">
       <property name="geometry">
        <rect>
         <x>67</x>
         <y>102</y>
         <width>591</width>
         <height>341</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 290 12pt &quot;微软雅黑 Light&quot;;</string>
       </property>
       <property name="text">
        <string>排片表：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="back">
       <property name="geometry">
        <rect>
         <x>301</x>
         <y>480</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>前往排片</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit_2">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>41</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="assign_search">
       <property name="geometry">
        <rect>
         <x>378</x>
         <y>35</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>查看该日期排片表</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="none_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
